### PR TITLE
LPD-25676  Give proper feedback to screen reader users when using Field Base

### PR DESCRIPTION
**Pull request by ilzamcmed**
### References
[LPD-25676 Jira issue](https://liferay.atlassian.net/browse/LPD-25676)

### What is the goal of this PR?
Make the feedback field component accessible for screen users and return a proper feedback message.

### What does it look like?

https://github.com/liferay-frontend/liferay-portal/assets/47724428/21804a20-b817-4567-a7ca-6bc645ffae31

### Before PR
When using a Screen Reader, if an error or warning message is displayed, the screen reader doesn't give a proper message and users have no idea of what's shown on the screen.

### After PR
After the error or warning message is displayed, the Screen Reader reads the feedback message and the user becomes aware of the feedback.

### Steps to reproduce

- Add a Screen Reader extension on Chrome;
- Navigate to Control panel > Client Extension > Plus Button > Add JS;
- Press the "tab" key until you focus on the attribute field;
- Type "src" on the input and you will see the error;
- Press "tab" to the next element and expect to hear a feedback message after the screen reader describe the element.

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/common/FieldFeedback.tsx [hunk 1]
```diff
@@ -14,27 +14,18 @@ export default function FieldFeedback({
 	warningMessage,
 	...otherProps
 }: IProps) {
-	if (!errorMessage && !helpMessage && !warningMessage) {
-		return null;
-	}
+	const showWarning = warningMessage && !errorMessage;
+
+	const message = showWarning ? warningMessage : errorMessage;
+	const symbol = showWarning ? 'warning-full' : 'exclamation-full';
```
Comment (markocikos) [inline modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/common/FieldFeedback.tsx:20]: Could we simplify this to:
```js
const message = errorMessage ? errorMessage : warningMessage;
const symbol = errorMessage ? 'exclamation-full' : 'warning-full';
```

File: modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/common/FieldBase.tsx [hunk 2]
```diff
@@ -121,6 +121,7 @@ export default function FieldBase({
 				<FieldFeedback
 					errorMessage={errorMessage}
 					helpMessage={helpMessage}
+					id={`${id}fieldFeedback`}
```
Comment (markocikos) [inline modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/common/FieldBase.tsx:124]: Why do we need to set `id`?

**Review by markocikos — COMMENTED** [review]
@ilzamcmed few questions inline

File: modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/common/FieldBase.tsx (hunk 2 above)
Comment (ilzamcmed) [reply modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/common/FieldBase.tsx:124]: We need the `id` here so the screen reader can make a relation to the `aria-describedby` and give a proper description of the component.

Comment (markocikos) [reply modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/common/FieldBase.tsx:124]: Got it, thanks for the clarification!

**Review by markocikos — APPROVED** [review]
@ilzamcmed LGTM! :sparkles:

**Pull request closed without merging**